10-Pull-To-Refresh/MainTableViewController.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  MainTableViewController.swift
+//  10-Pull-To-Refresh
+//
+//  Created by vmalikov on 3/4/16.
+//  Copyright © 2016 justForFun. All rights reserved.
+//
+
+import UIKit
+
+class MainTableViewController: UITableViewController {
+    
+    var dataSource = ["Milk", "Apples", "Ham", "Eggs"]
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        self.clearsSelectionOnViewWillAppear = false
+        
+        addRefreshControl()
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+    }
+    
+    func addRefreshControl() {
+        if self.refreshControl == nil {
+            self.refreshControl = UIRefreshControl()
```

Comment:
No need to do this in the `if` statement as this is called from viewDidLoad, so it will be called just once.